### PR TITLE
Update commercial_support.rst

### PR DESCRIPTION
**Pull request by amaclenan**
Updated the company infomration for Astun

### CODE REVIEW

File: source/site/forusers/commercial_support.rst [hunk 1]
```diff
@@ -231,8 +231,9 @@ Others
 .. |astun| image:: /static/site/about/images/astun.jpg
    :width: 75 px
 
-|astun| `Astun Technology <http://astuntechnology.com/>`_ (based in UK) provides
-training for QGIS.
+|astun| `Astun Technology <http://astuntechnology.com/>`_ was formed in 2005 to provide open source and web-based GIS services to local and central governments.  Today, as all organisations realise the technical and financial benefits of using cloud solutions to create, manage and deliver their crucial GEO services, the demand for Astun's knowledge and experience in delivering marketing leading, fully managed AWS-based GIS cloud solutions, continues to grow. 
+For more information, please visit https://astuntechnology.com/"
```
Comment (yjacolin) [inline source/site/forusers/commercial_support.rst:235]: you should remove the final *"*

**Review by yjacolin — CHANGES_REQUESTED** [review]

File: source/site/forusers/commercial_support.rst (hunk 1 above)
Comment (amaclenan) [reply source/site/forusers/commercial_support.rst:235]: Thanks. Hopefully I have done this correctly but I have created a new pull request?

Comment (yjacolin) [reply source/site/forusers/commercial_support.rst:235]: you can push to/edit this branch to update the pull request.

**Comment by yjacolin** [comment]
See #443

**Pull request closed without merging**